test/test-integration-3rd/src/test/java/com/alipay/sofa/rpc/bootstrap/dubbo/DubboServerTest.java

```diff
@@ -42,6 +44,9 @@
     @Test
     //同步调用,走服务注册中心
     public void testRegistrySync() {
+        System.setProperty(ConfigKeys.DUBBO_CONFIG_IGNORE_DUPLICATED_INTERFACE, "true");
```

Comment:
It is best to restore the System properties settings after this test case is completed